pkg/entities/entity_integration_test.go

```diff
@@ -238,20 +248,24 @@ func TestIntegrationGetEntity_MobileEntity(t *testing.T) {
 
 	client := newIntegrationTestClient(t)
 
-	result, err := client.GetEntity("MzgwNjUyNnxNT0JJTEV8QVBQTElDQVRJT058NjAxMzc1OTAx")
+	result, err := client.GetEntity("MzgwNjUyNnxNT0JJTEV8QVBQTElDQVRJT058NjAxNDQ1MTYx")
 
 	if e, ok := err.(*http.GraphQLErrorResponse); ok {
 		if !e.IsDeprecated() {
 			require.NoError(t, e)
 		}
 	}
-	require.NotNil(t, (*result))
 
+	if *result == nil {
+		t.Skipf("Skipping this test as MobileApplicationEntities are fragile, need to be recreated")
+	}
+
+	require.NotNil(t, (*result))
 	actual := (*result).(*MobileApplicationEntity)
 
 	// These are a bit fragile, if the above GUID ever changes...
 	// from MobileApplicationEntity / MobileApplicationEntityOutline
-	assert.Equal(t, 601375901, actual.ApplicationID)
+	assert.Equal(t, 601445161, actual.ApplicationID)
```

Comment:
`func TestIntegrationGetEntity_MobileEntity(t *testing.T)`
- since the older GUID of the Mobile Entity (which has been in use so far) is obsolete and can no longer be found in the New Relic account, _temporarily_ changed the GUID to that of a new Mobile Entity
- however, since instrumented mobile entities are fragile too (as we don't follow a regular APM-type procedure), added a `Skip` method to not report a failure, if the Mobile Entity with this GUID ceases to exist some day